tests/test_something.py

```diff
@@ -68,7 +68,7 @@ def start(self):
                 "HOME": str(self.home_path),
                 "addr": self.addr,
                 "mail_pw": self.password,
-                "RUST_LOG": "xdcstore=trace",
+                "RUST_LOG": "xdcstore=info",
```

Comment:
For me, `trace` is waaaay to spammy with not enough information. I'd prefer to keep it as this and to only change back to trace locally if it is needed.